src/node_crypto.cc

```diff
@@ -1641,8 +1641,23 @@ void SSLWrap<Base>::OnClientHello(void* arg,
                                              hello.servername_size());
     hello_obj->Set(env->servername_string(), servername);
   }
-  hello_obj->Set(env->tls_ticket_string(),
-                 Boolean::New(env->isolate(), hello.has_ticket()));
+
+  // Overload ticket to be a false boolean if not defined and the actual
+  // ticket IV if defined
+  if (hello.tls_ticket() == nullptr) {
+    hello_obj->Set(env->tls_ticket_string(),
+                Boolean::New(env->isolate(), hello.has_ticket()));
+  } else {
+    size_t key_name_offset = 16;
+    // IV is the only part we are actually going to take
+    size_t key_iv_size = 16;
+    buff = Buffer::Copy(
```

Comment:
We strip the IV from the session ticket and expose it as part of the ClientHello object that propagates to JS land